test/config/config.js

```diff
@@ -28,6 +28,11 @@ module.exports = {
     password: process.env.SEQ_MSSQL_PW   || process.env.SEQ_PW   || null,
     host:     process.env.SEQ_MSSQL_HOST || process.env.SEQ_HOST || '127.0.0.1',
     port:     process.env.SEQ_MSSQL_PORT || process.env.SEQ_PORT || 11433,
+    dialectOptions: {
+      instanceName:  process.env.MSSQL_INSTANCE ? process.env.MSSQL_INSTANCE : 'SQLEXPRESS',
+      // big insert queries need a while
```

Comment:
@TimMurnaghan Could you please remove the `instanceName` from dialect options? I can't connect to my local sql server and it doesn't default to `SQLEXPRESS`. Why not directly connect to your server via `host` and `port`?